pkg/controllers/keymanagementprovider_controller.go

```diff
@@ -80,9 +80,8 @@ func (r *KeyManagementProviderReconciler) Reconcile(ctx context.Context, req ctr
 	}
 	// if certificate store is configured, return error. Only one of certificate store and key management provider can be configured
 	if len(certificateStoreList.Items) > 0 {
-		err := re.ErrorCodeKeyManagementConflict.WithComponentType(re.KeyManagementProvider).WithPluginName(resource).WithDetail("certificate store already exists")
-		// Note: for backwards compatibility in upgrade scenarios, Ratify will only log an error.
-		logger.Error(err)
+		// Note: for backwards compatibility in upgrade scenarios, Ratify will only log a warning statement.
+		logger.Warn("Certificate Store already exists. Key management provider and certificate store should not be configured together. Please migrate to key management provider and delete certificate store.")
```

Comment:
curious if we should update the status in this case

Reply:
yeah that's a good point. I originally was updating the status. The problem is during remediation. Let's say a user follows the warning in the logs and then deletes the certificate store, the status on the KMP resource does not get updated automatically since the controller for KMP isn't triggered again